little_cheesemonger/_run.py

```diff
@@ -66,10 +69,18 @@ def install_system_dependencies(dependencies: List[str]) -> None:
     run_subprocess(["yum", "install", "-y"] + dependencies)
 
 
-def install_python_dependencies(dependencies: List[str]) -> None:
+def install_python_dependencies(
+    dependencies: List[str], python_versions: Optional[List[str]]
```

Comment:
Consider adding default value

```suggestion
    dependencies: List[str], python_versions: Optional[List[str]] = None
```